var/spack/repos/builtin/packages/diamond/package.py

```diff
@@ -14,7 +14,7 @@ class Diamond(CMakePackage):
     url      = "https://github.com/bbuchfink/diamond/archive/v0.9.14.tar.gz"
 
     version('2.0.4', sha256='94e8fe72bdc28b83fd0f2d90c439b58b63b38263aa1a3905582ef68f614ae95d',
-            url="https://github.com/bbuchfink/diamond/archive/diamond-2.0.4.tar.gz")
+            url="https://github.com/bbuchfink/diamond/archive/v2.0.4.tar.gz")
```

Comment:
In that case we no longer need to specify the URL at all, the default URL will work.

Reply:
I removed the v2.0.4 URL.